Source/WebCore/html/parser/ParsingUtilities.h

```diff
@@ -213,7 +213,7 @@ template<typename CharacterType> bool skipExactlyIgnoringASCIICase(const Charact
     
     if (position + literalLength > end)
         return false;
-    if (!equalLettersIgnoringASCIICase(position, literal))
+    if (!equalLettersIgnoringASCIICaseWithLength(std::span { position, literalLength }, literal.span8(), literalLength))
```

Comment:
I don’t understand why `literalLength` has to be passed here.

Reply:
If we call equalLettersIgnoringASCIICase() instead of equalLettersIgnoringASCIICaseWithLength(), it will check that the lengths of the 2 parameters are equal. This check would be redundant here.